perllib/FixMyStreet/Geocode/Google.pm

```diff
@@ -19,11 +19,13 @@ sub string {
 
     my $params = $c->cobrand->disambiguate_location($s);
 
+    my $components = "";
+
     # For some reason adding gl=uk is no longer sufficient to make google
-    # think we are in the UK for some locations so we explictly add UK to
-    # the address.
+    # think we are in the UK for some locations so we explicitly tell Google
+    # the country.
     if ($c->cobrand->country eq 'GB' && $s !~ /, *UK/ && $s !~ /united *kingdom$/) {
-        $s .= ', UK';
+        $components = "country:GB";
```

Comment:
Do we want to always add this if the cobrand country is GB (ie drop the rest of the if statement above)?